src/payload-receiver-service/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8
```

Comment:
a suggestion that apply to all of our images:
for images that the usage is as lightweight as this one, we should probably use python:3.8-slim instead.

Reply:
using `python:3.8-slim ` now